docs/en/latest/plugins/inspect.md

```diff
@@ -0,0 +1,171 @@
+
+## Operation Graph
+
+![Operation Graph](https://raw.githubusercontent.com/apache/apisix/master/docs/assets/images/plugin/inspect.png)
+
+## API to define hook in hooks file
+
+### require("resty.inspect.dbg").set_hook(file, line, func, filter_func)
+
+The breakpoint is specified by `file` (full qualified or short file name) and the `line` number.
+
+The `func` specified the scope (which function or global) of jit cache to flush:
+
+* If the breakpoint is related to a module function or
+global function, you should set it that function reference, then only the jit cache of that function would
+be flushed, and it would not affect other caches to avoid slowing down other parts of the program.
+
+* If the breakpointis related to local function or anonymous function,
+then you have to set it to `nil` (because no way to get function reference), which would flush the whole jit cache of Lua vm.
+
+You attach a `filter_func` function of the breakpoint, the function takes the `info` as argument and returns
+true of false to determine whether the breakpoint would be removed. You could setup one-shot breakpoint
+at ease.
+
+The `info` is a hash table which contains below keys:
+
+* `finfo`: `debug.getinfo(level, "nSlf")`
+* `uv`: upvalues hash table
+* `vals`: local variables hash table
+
+## Attributes
+
+| Name               | Type    | Required | Default | Description                                                                                    |
+|--------------------|---------|----------|---------|------------------------------------------------------------------------------------------------|
+| delay           | integer | False     | 3 | Time in seconds specifying how often to check the hooks file.                                       |
+| hooks_file           | string | False     | "/var/run/apisix_inspect_hooks.lua"  | Lua file to define hooks, which could be a link file. |
+
+## Enabling the Plugin
+
+Plugin is enabled by default (`conf/config-default.yaml`):
+
+```yaml title="conf/config-default.yaml"
+plugins:
+    - inspect
+
+plugin_attr:
+  inspect:
+    delay: 3
+    hooks_file: "/var/run/apisix_inspect_hooks.lua"
+```
+
+## Example usage
+
+```bash
+# create test route
+curl http://127.0.0.1:9180/apisix/admin/routes/test_limit_req -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "methods": ["GET"],
+    "uri": "/get",
+    "plugins": {
+        "limit-req": {
+            "rate": 100,
+            "burst": 0,
+            "rejected_code": 503,
+            "key_type": "var",
+            "key": "remote_addr"
+        }
+    },
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "httpbin.org": 1
+        }
+    }
+}'
+
+# create a hooks file to set a test breakpoint
+# Note that the breakpoint is associated with the line number,
+# so if the Lua code changes, you need to adjust the line number in the hooks file
+cat <<EOF >/tmp/hooks.lua
```

Comment:
Do not use the /tmp directory, especially for documentation

Reply:
ok, I would change it.